packages/create-cloudflare/templates/astro/c3.ts

```diff
@@ -14,7 +14,7 @@ const generate = async (ctx: C3Context) => {
 	logRaw(""); // newline
 };
 
-const configure = async (ctx: C3Context) => {
+const configure = async (_ctx: C3Context) => {
```

Comment:
Similarly this could be deleted.

Reply:
Here I left it intentionally

This `configure` function doesn't have a specific type/signature but it does receive a `C3Context` when getting called, this is something that might not be fully apparent to people new to the codebase and I thought that @jculvey could have left the parameter there indeed to make the code more self-documenting

I'm totally happy with removing it if you prefer, do you?

Reply:
It does have a specific type in the `TemplateConfig`. So if you created a weird function here it would error when trying to assign it to `TemplateConfig.configure`.
Personally I would just remove it.

Reply:
I don't think it's a huge deal either way, removed 🙂👍